contracts/test/mocks/LightClientTest.sol

```diff
@@ -9,9 +9,13 @@ import { LightClientStateUpdateVKTest as VkLib } from "./LightClientStateUpdateV
 
 /// @dev A helper that wraps LightClient contract for testing
 contract LightClientTest is LC {
-    constructor(LC.LightClientState memory genesis, uint32 numBlockPerEpoch)
-        LC(genesis, numBlockPerEpoch)
-    { }
+    constructor() LC() { }
+
+    function initializeForTests(LC.LightClientState memory genesis, uint32 numBlockPerEpoch)
+        public
+    {
+        _initializeState(genesis, numBlockPerEpoch);
+    }
```

Comment:
why couldn't we call `_initializeState` inside constructor here?
I know that LightClient couldn't call constructor, but this contract can, right?

this is only used for non-upgradabed tests only anyway.

Reply:
Good catch. Change done in 135b6a1.